x-pack/plugins/enterprise_search/server/lib/indices/generate_api_key.ts

```diff
@@ -18,7 +18,7 @@ export const generateApiKey = async (client: IScopedClusterClient, indexName: st
         cluster: [],
         index: [
           {
-            names: [indexName, CONNECTORS_INDEX],
+            names: [indexName, `${CONNECTORS_INDEX}*`],
```

Comment:
Question: why do we create with wildcards?

Reply:
Ah, because we now have a jobs index and are likely to add a few more as we develop this framework. We're essentially claiming this namespace.